### DISCUSSION TITLE
Is there any way to commit by my github username instead of github bot ?

### DISCUSSION THREAD

**Discussion by msrajawat298** [gautamkrishnar/blog-post-workflow#275] [Q&A] [answered]
name: Latest blog post workflow
on:
  schedule: # Run workflow automatically
    - cron: '0 0-23 * * *' # Runs every hour, on the hour
  workflow_dispatch: # Run workflow manually (without waiting for the cron to be called), through the GitHub Actions Workflow page directly
permissions:
  contents: write # To write the generated contents to the readme

jobs:
  update-readme-with-blog:
    name: Update this repo's README with latest blog posts
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v3
      - name: Pull in Blog posts
        uses: gautamkrishnar/blog-post-workflow@v1
        with:
          feed_list: 'https://blog.vitabletech.in/feed/'

**Comment by gautamkrishnar** [comment, marked as answer]
use `committer_username` and `committer_email` parameters. Check https://github.com/gautamkrishnar/blog-post-workflow?tab=readme-ov-file#options
